stable/mongodb-replicaset/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: mongodb-replicaset
 home: https://github.com/mongodb/mongo
-version: 3.9.6
+version: 3.9.7
```

Comment:
This is definitely a breaking change which would need a major bump.

Reply:
Will adjust